stuffer/s2n_stuffer.c

```diff
@@ -39,7 +39,7 @@ int s2n_stuffer_init(struct s2n_stuffer *stuffer, struct s2n_blob *in)
     return 0;
 }
 
-int s2n_stuffer_alloc(struct s2n_stuffer *stuffer, uint32_t size)
+int s2n_stuffer_alloc(struct s2n_stuffer *stuffer, const uint32_t size)
```

Comment:
does this mean that:

uint32_t x = 5; // not a const
s2n_stuffer_alloc(stuffer, x);

would not work? What does a compiler do with const in the pass-by-value case?